cdap-ui/app/cdap/components/PipelineSummary/Store/PipelineSummaryActions.js

```diff
@@ -84,20 +85,23 @@ function updateMetrics(metrics, mapKey = 'metrics') {
 
 function updateNodeMetrics(pipelineConfig) {
   let {runs} = PipelineSummaryStore.getState().pipelinerunssummary;
-  const sourcePluginTypes = ['batchsource', 'realtimesource', 'streamingsource'];
-  const sinkPluginTypes = ['batchsink', 'realtimesink', 'sparksink'];
-
   let nodesMap = {
-    sources: {},
-    sinks: {}
+    recordsin: {},
+    recordsout: {}
   };
+  const sourcePluginTypes = ['batchsource', 'realtimesource', 'streamingsource'];
+  const sinkPluginTypes = ['batchsink', 'realtimesink', 'sparksink'];
+  const sourcePlugins = [];
+  const sinkPlugins = [];
```

Comment:
`sourcePlugins` and `sinkPlugins` should be initialized with `let`, since we're modifying them?